Adjusting keepAliveTimeout for the moleculer-web server (#1785)
LiskHQ/lisk-service: services/gateway/README.md

- `ENABLE_WS_API`: Enables particular JSON-RPC APIs listed by a comma.
- `CORS_ALLOWED_ORIGIN`: Allows request from the comma separated string of origins. By default, it is set to `*` which allows request from all origins.
- `SERVICE_GATEWAY_REDIS_VOLATILE`: URL of the volatile cache storage (Redis).

> - `ENABLE_REVERSE_PROXY_CONNECTION`: Boolean flag to indicate presence of a reverse proxy or a load balancer between gateway and the client.
> - `HTTP_KEEP_ALIVE_TIMEOUT`: Defines the number of seconds gateway will wait before closing an idle connection. To enable, ensure `ENABLE_REVERSE_PROXY_CONNECTION` is set to true or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. By default, it is set to 65000.
> - `HTTP_HEADERS_TIMEOUT`: Defines the maximum number of seconds for gateway to send HTTP response headers after the client's request. Ensure the headersTimeout is set higher than the keepAliveTimeout. By default, it is set to 66000.

Reviewer: ```suggestion
- `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS`: Boolean flag to enforce the `headersTimeout` and `keepAliveTimeout` settings on the API server. Enabling this might be helpful when deploying Lisk Service behind a load balancer or a reverse proxy. Check [this FAQ](https://moleculer.services/docs/0.14/faq.html#Why-am-I-getting-502-Bad-Gateway-when-api-gateway-is-behind-ALB-on-AWS) for more information.
- `HTTP_KEEP_ALIVE_TIMEOUT`: Defines the number of microseconds gateway will wait before closing an idle connection. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. By default, it is set to 65000.
- `HTTP_HEADERS_TIMEOUT`: Defines the maximum number of microseconds for gateway to send HTTP response headers after the client's request. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. Please ensure that the `HTTP_HEADERS_TIMEOUT` is set higher than the `HTTP_KEEP_ALIVE_TIMEOUT`. By default, it is set to 66000.
```

> - `HTTP_KEEP_ALIVE_TIMEOUT`: Defines the number of microseconds gateway will wait before closing an idle connection. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. By default, it is set to 65000.

Reviewer 2: ```suggestion
- `HTTP_KEEP_ALIVE_TIMEOUT`: Defines the number of microseconds the gateway will wait before closing an idle connection. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true, or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. By default, it is set to 65000.
```

> - `HTTP_HEADERS_TIMEOUT`: Defines the maximum number of microseconds for gateway to send HTTP response headers after the client's request. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. Please ensure that the `HTTP_HEADERS_TIMEOUT` is set higher than the `HTTP_KEEP_ALIVE_TIMEOUT`. By default, it is set to 66000.

Reviewer 2: ```suggestion
- `HTTP_HEADERS_TIMEOUT`: Defines the maximum number of microseconds for the gateway to send HTTP response headers after the client's request. To enable, ensure `ENABLE_REVERSE_PROXY_TIMEOUT_SETTINGS` is set to true, or `HTTP_RATE_LIMIT_NUM_KNOWN_PROXIES` is set to greater than 0. Please ensure that the `HTTP_HEADERS_TIMEOUT` is set higher than the `HTTP_KEEP_ALIVE_TIMEOUT`. By default, it is set to 66000.
```
